index.html

```diff
@@ -139,10 +146,12 @@ <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
     <section class="about-me"> 
         <div class="about-text">
             <h3 class="p-text">About me</h3>
+       
             <p class="s-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                 don’t hestiate to contact me.</p>
+    
                 <button id="resume-button" type="button">Get My Resume</button>
-        </div>
+            </div>
         <div class="about-card">
             <div class="language-card">
```

Comment:
- [ ] Kindly notice that the about me section is also a little bit to the left, please fix that.

Your Desing | Figma Design
---------------- | -------------------
![Screenshot from 2022-08-25 16-43-22](https://user-images.githubusercontent.com/33752735/186698650-94811356-1896-4588-9e69-2937bbed0b29.png) | ![Screenshot from 2022-08-25 16-54-52](https://user-images.githubusercontent.com/33752735/186698894-4ee3b599-84f7-4a34-94f8-2d44cfba56ba.png)
![Screenshot from 2022-08-25 16-43-51](https://user-images.githubusercontent.com/33752735/186698950-97f4034a-b683-4175-b885-0bcdec260c31.png) | ![Screenshot from 2022-08-25 16-55-19](https://user-images.githubusercontent.com/33752735/186698996-69cb1fca-dad0-4634-b2b2-ba14c27efa87.png)

Reply:
- [ ] Please align these cards as requested by the previous reviewer since it is a core requirement for you to follow the design seen on the Figma template 👍